lanelet2/lanelet2_routing/src/RoutingGraphBuilder.cpp

```diff
@@ -285,10 +285,13 @@ void RoutingGraphBuilder::addAreaEdge(const ConstArea& area, const LaneletLayer&
       continue;
     }
     auto maxHeight = participantHeight();
+#if 0
+    // TODO(mjeronimo): fix the overlaps2d/3d issue
     if ((maxHeight && geometry::overlaps3d(area, candidate, *maxHeight)) ||
         (!maxHeight && geometry::overlaps2d(area, candidate))) {
       assignCosts(candidate, area, RelationType::Conflicting);
     }
+#endif
```

Comment:
Edits to this file could definitely have a negative impact, so it looks like the overlaps2d/3d is probably something we need to address.

Reply:
Enabled.